### PR TITLE
Fix handling of files with spaces in frontend_server_client

### PR DESCRIPTION
**Pull request by jakemac53**
Fixes https://github.com/dart-lang/webdev/issues/1380﻿

Also enables running of the tests on windows/linux.

### CODE REVIEW

**Comment by jakemac53** [comment]
This should hopefully fail on windows and I can start diagnosing things.

**Comment by jakemac53** [comment]
Nice, this test repros on linux as well 👍. You only see it on windows in practice because spaces in file/directory names are pretty unique to windows.

File: frontend_server_client/mono_pkg.yaml [hunk 1]
```diff
@@ -4,4 +4,12 @@ stages:
...
-      dart: [2.12.0]
+      dart: dev
+  - unit_test:
+    - test:
+      dart:
+        - stable
+        - dev
+      os:
+        - windows
+        - linux
```
Comment (natebosch) [inline frontend_server_client/mono_pkg.yaml:15]: [nit] Add a trailing newline

**Review by natebosch — APPROVED** [review]

**Comment by jakemac53** [comment]
cc @jonasfj @sigurdm You may want to add some tests in pub for this case as well? Just to ensure a similar issue doesn't crop up if the compilation strategy were to change in the future.

**Pull request merged**